app/presenters/electoral_presenter.rb

```diff
@@ -44,6 +45,12 @@ def present_ballots(date)
     end
   end
 
+  def present_addresses
+    if addresses.present?
+      addresses.map { |a| "<a href='#{find_electoral_things_path}?uprn=#{a['slug']}'>#{a['address']}</a>" }
```

Comment:
Might be nicer to pass through the details and iterate using a `link_to` in the view?